composer.json

```diff
@@ -10,7 +10,8 @@
     "cakephp/plugin-installer": "*"
   },
   "require-dev": {
-    "phpunit/phpunit": "4.1.*"
+    "phpunit/phpunit": "4.1.*",
+    "loadsys/loadsys_codesniffer": "~3.0"
```

Comment:
I added it due to these lines: https://github.com/loadsys/CakePHP-Skeleton/pull/117/files#diff-04c6e90faac2675aa89e2176d2eec7d8L150